deb/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: sdlhmi
+Version: 6.1.0
+Provides: sdlhmi
+Maintainer: Alexander Kutsan <akutsan@luxoft.com>
```

Comment:
@LuxoftAKutsan not sure its a good practice to leave own email in the opensource repository. Are you really going to maintain this?

Reply:
Fixed